MobileWallet/Assets.xcassets/Colors/HomeBackground.colorset/Contents.json

```diff
@@ -9,10 +9,10 @@
       "color" : {
         "color-space" : "srgb",
         "components" : {
-          "red" : "0.897",
+          "red" : "229",
```

Comment:
Is there an advantage to changing all the colours to 8-bit?

Reply:
It's easier to read it from Zeplin mainly. No other advantage I know off technically.

Reply:
cool